pytrends/test_trendReq.py

```diff
@@ -85,6 +85,15 @@ def test_realtime_trending_searches(self):
         pytrend.build_payload(kw_list=['pizza', 'bagel'])
         self.assertIsNotNone(pytrend.realtime_trending_searches(pn='IN'))
 
+    def test_request_args_passing(self):
+        requests_args = {'headers': {
+            'User-Agent': 'pytrends',
+        }}
+        pytrend = TrendReq(requests_args=requests_args)
+        pytrend.build_payload(kw_list=['bananas'])
```

Comment:
I think this call is not needed, but other tests have it, so I left it here too.